src/types.ts

```diff
@@ -161,6 +166,8 @@ export interface EventQuery {
   activeAtTimeRange: boolean;
   columns: string[];
   advancedFilter: string;
+  withAggregate: boolean;
+  property: string[];
```

Comment:
I suggest having this as a nested object:
```suggestion
  aggregate?: {
         type: "uniqueValues" | "count",
         properties: { property: string }[]
  }
```